torch/_inductor/codegen/cpp_wrapper_cpu.py

```diff
@@ -1717,7 +1721,11 @@ def create_dtypeview_call(reinterpret_call):
                 )
                 call_strs = [f"AtenTensorHandle {tmp_AtenTensorHandle};"]
                 dtype_name = str(dtype).split(".")[-1]
-                device_name = "cuda" if data.layout.device.type == "cuda" else "cpu"
+                device_name = (
+                    data.layout.device.type
+                    if is_gpu(data.layout.device.type)
+                    else "cpu"
```

Comment:
why not just unconditionally data.layout.device.type here?

Reply:
> why not just unconditionally data.layout.device.type here?

Sure, the origin `device_name = "cuda" if data.layout.device.type == "cuda" else "cpu"` also confused me.